tests/BuildScriptGeneratorCli.Tests/BuildCommandTest.cs

```diff
@@ -10,11 +10,10 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.DependencyInjection.Extensions;
 using Microsoft.Oryx.BuildScriptGenerator;
-using Microsoft.Oryx.BuildScriptGeneratorCli;
 using Microsoft.Oryx.Tests.Common;
 using Xunit;
 
-namespace BuildScriptGeneratorCli.Tests
+namespace Microsoft.Oryx.BuildScriptGeneratorCli.Tests
```

Comment:
👍  Make sure to have similar settings for this project's .csproj file too...this way any new files would have these changes..

```
    <AssemblyName>Microsoft.Oryx.BuildScriptGenerator.Tests</AssemblyName>
    <RootNamespace>Microsoft.Oryx.BuildScriptGenerator.Tests</RootNamespace>
```